docs/introduction/get-started/p2p-private.md

```diff
@@ -32,7 +32,7 @@ In this section, we show you how to make off-chain transactions and send funds t
     To do this, run:
 
     ```shell
-    miden-client tx new p2id <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 --note-type private
+    miden tx new p2id <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 --note-type private
```

Comment:
Same here, do we need to add flags here?

https://github.com/0xPolygonMiden/miden-client/blob/next/docs/cli-reference.md#types-of-transaction